cargo-concordium/README.md

```diff
@@ -62,6 +62,90 @@ or even `opt-level = "z"`.
 
 In some cases using `opt-level=3` actually leads to smaller code sizes, presumably due to more inlining and dead code removal as a result.
 
+## Reproducible and verifiable builds.
+
+A normal build with `cargo concordium build` is generally not reproducible since
+some host information is embedded into the resulting binary. For this reason
+`cargo concordium` supports so-called verifiable or reproducible builds that
+always build the contract in a fixed environment specified as a Docker image.
+The list of available images can be found on
```

Comment:
I don't think that makes sense for cargo-concordium.

It would make sense for ccdscan when putting checkmarks about "verified" though. It should not just allow arbitrary images.

But I think disallowing other images here is not in the spirit of decentralization.

It makes sense to point out the trust anchors though.

Reply:
I added a sentence.